docs/paper/admin/reference/configuration/global-configuration.mdx

```diff
@@ -0,0 +1,403 @@
+---
+slug: /reference/global-configuration
+---
+
+# Global Configuration
+
+import React from 'react';
+import Config from '../../../../../src/components/Config';
```

Comment:
Should be able to use the `@site` [prefix](https://docusaurus.io/docs/markdown-features/react#importing-components) instead of the massive chain of `..`.